src/libs/API.js

```diff
@@ -621,6 +620,17 @@ function Mobile_GetConstants(parameters) {
     return Network.post(commandName, finalParameters);
 }
 
+/**
+ * @param {Object} parameters
+ * @param {String} parameters.debtorEmail
+ * @returns {Promise}
+ */
+function GetIOUReport(parameters) {
+    const commandName = 'GetIOUReport';
```

Comment:
Hm, I would have really liked for this API to be called `GetIOUReportID` since it's only returning the report ID and nothing else. It would be even better if this API would remain named `GetIOUReport` and return the entire report object that is later requested in a second API request (get stuff).

Reply:
hmmm maybe yeah, let's keep as is and we can do that as an optimization later.

Reply:
No, I don't think this is too much of a pre-optimization to ask for. This isn't about optimizing, this is about doing it the correct way in the first place. We should never settle for making two API calls when only one API call will do. Since this is building a new feature, let's do it correctly rather than to force ourselves to have future work.

Reply:
ok, sending a PR that's backward compatible with this. We can wait for it and use it in this PR or probably better, merge this as is to unblock other issues and use the new data in a followup PR.

Reply:
I'm OK implementing it in another PR.

Reply:
PRs here:
https://github.com/Expensify/Web-Expensify/pull/30479 
https://github.com/Expensify/Auth/pull/5368

Reply:
Once https://github.com/Expensify/Web-Expensify/pull/30479 hits production I can create a follow-up for this

Reply:
This is not live, so we can do that now I think...

Reply:
After some testing, it looks like the changes in https://github.com/Expensify/Web-Expensify/pull/30479 aren't returning the transactionList, so we shouldn't update this to remove the GetReportStuff call yet. Once I've updated GetIOUReport to also return the transactionList (looking into it), then I can create a follow-up here.

Reply:
Issue created here: https://github.com/Expensify/Expensify/issues/157722

Reply:
Looks like this thread can be resolved